crates/aptos/src/common/types.rs

```diff
@@ -1246,6 +1246,11 @@ pub trait CliCommand<T: Serialize + Send>: Sized + Send {
     /// Returns a name for logging purposes
     fn command_name(&self) -> &'static str;
 
+    /// Returns whether the error should be JSONifyed.
+    fn jsonify_error_output(&self) -> bool {
+        true
+    }
```

Comment:
Maybe I am missing something. Since this is hardcoded to true, how do I opt out to not jsonify the output?

Reply:
This is the default trait implementation, any CLI command can make this return false instead. This is what I do in the local testnet subcommand.

Reply:
neato!